<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateOptionDialogWidget</class>
 <widget class="QWidget" name="DateOptionDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>534</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,10">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,10,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string comment="the source of the date (image, system etc...)">Source:</string>
       </property>
       <property name="buddy">
        <cstring>dateFormatPicker</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="dateSourcePicker"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string comment="the format of the date">Format:</string>
       </property>
       <property name="buddy">
        <cstring>dateFormatPicker</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="dateFormatPicker"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="customFormatInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="exampleLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string notr="true" comment="KDE::DoNotExtract">placeholder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dateFormatLink">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true" comment="KDE::DoNotExtract">placeholder</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="fixedDateContainer" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="Digikam::DDatePicker" name="datePicker"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timePicker"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::DDatePicker</class>
   <extends>QFrame</extends>
   <header>ddatepicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateSourcePicker</tabstop>
  <tabstop>dateFormatPicker</tabstop>
  <tabstop>timePicker</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
